qt_touchup_app.py

```diff
@@ -132,21 +132,22 @@ def load_img_and_render(self):
 
         try:
             self.img_raw, self.w, self.h = qtl.load_img(self.imgpath)
+        except:
```

Comment:
changes here are just reducing the size of the try/except block so future errors aren't buried